src/pages/SearchPage.tsx

```
@@ -0,0 +1,58 @@
+            console.error('Failed to navigate:', error);
+        }
+    };
+
+    const inputValid = stationInfoContext.crsList.includes(originStation) &&
+        stationInfoContext.crsList.includes(destinationStation) &&
+        originStation !== destinationStation;
+
+    return (
+        <div>
+            <h1 id = "search-title">Find your next journey...</h1>
+            <main role = "main" aria-labelledby = "search-title">
```

Biome

[error] The elements with the following roles can be changed to the following elements:
<main>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)